portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,12 +26,40 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
  
-  private String greeting;
+  private List<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+    comments.add("This is a nice page.");
+    comments.add("You like engineering? I do too.");
+    comments.add("Cool website.");
+  }
+
  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    greeting = "Hello Caleb";
-    response.setContentType("text/html;");
-    response.getWriter().println(greeting);
+    String json = convertToJson(comments);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+    /**
+   * Converts a list of comments into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(List<String> comments) {
```

Comment:
The hard coded string implementation can be replaced by Gson

https://stackoverflow.com/questions/14228912/how-to-convert-list-to-json-in-java

Reply:
Okay, I will change in anew pull request.